### DISCUSSION TITLE
恢复备份后重启权限管理和移花接木被关闭，情景模式规则都读取不出来

### DISCUSSION THREAD

**Discussion by jinbolin227** [Tornaco/Thanox#322] [General]
**Describe the bug（现象描述）**
A clear and concise description of what the bug is.（简洁的描述该bug的现象）
恢复备份后重启权限管理和移花接木被关闭，情景模式规则都读取不出来
**To Reproduce**（如何复现）
在一加8T氢OS系统上没问题换到氧OS系统上就会出现
Steps to reproduce the behavior:（复现步骤）
1. Go to '...'（进入XXX）
2. Click on '....'（点击XXX）
3. Scroll down to '....'（下滑到XXX）
4. See error（发现错误XXX）

**Expected behavior**（期望结果）
希望得以修复
A clear and concise description of what you expected to happen.（描述一下期望的结果）

**Screenshots**（截图）
![Screenshot_20210906-150623](https://user-images.githubusercontent.com/56496767/132176405-d1cd4cb4-c5ba-4189-b441-9b54c5403c37.jpg)
![Screenshot_20210906-150609](https://user-images.githubusercontent.com/56496767/132176412-9ab46d97-3bff-4317-a3c0-7e9a7b231f98.jpg)
![Screenshot_20210906-150616](https://user-images.githubusercontent.com/56496767/132176420-8290fcd4-a897-44fa-9691-8deb8e70ad25.jpg)

If applicable, add screenshots to help explain your problem.（通过截图更好的解释你的问题）

**Version（发现问题的版本）:**
 - OS: [e.g. Android 11.0]（系统版本，例如Android 
 - Version [e.g. v3.0.0]（软件版本，例如v2.3.4）

**Additional context**（其他信息）
在2.5上也出现过
Add any other context about the problem here.（其他的补充信息）

** Logs**（日志文件）
务必将截取到的日志文件发到该issue里。
[thanox-core.log](https://github.com/Tornaco/Thanox/files/7114042/thanox-core.log)
[thanox-init.log](https://github.com/Tornaco/Thanox/files/7114043/thanox-init.log)
[2021-09-06.txt](https://github.com/Tornaco/Thanox/files/7114046/2021-09-06.txt)

**Comment by Tornaco** [comment]
@jinbolin227 手动删除data/system/thanos文件夹，重启重新配置试试，可能是该文件夹的权限或mode被其他进程修改了。

**Comment by jinbolin227** [comment]
> @jinbolin227 手动删除data/system/thanos文件夹，重启重新配置试试，可能是该文件夹的权限或mode被其他进程修改了。

我测试还是这样显示，都搞不明白这是为什么，氢就没事，氧就这样

**Comment by jinbolin227** [comment]
应该是不能用恢复，恢复就会这样，

**Comment by jinbolin227** [comment]
> @jinbolin227 手动删除data/system/thanos文件夹，重启重新配置试试，可能是该文件夹的权限或mode被其他进程修改了。

后台启动规则不能导入导出，移花接木导出规则后shi空的
![Screenshot_20210912-223205](https://user-images.githubusercontent.com/56496767/132991706-33430210-ae97-4426-8db4-3e52d3af0406.jpg)

**Comment by jinbolin227** [comment]
手动重新添加后，显示移花接木和权限管理打开了但实际没有效果，情景模式添加规则也是显示但不生效

**Comment by 0honus0** [comment]
同有这个问题

**Comment by 0honus0** [comment]
手动修改文件夹权限解决了

**Comment by Tornaco** [comment]
OK，感谢反馈，暂时还是这个解决方案，暂时关闭。

**Comment by jinbolin227** [comment]
> 手动修改文件夹权限解决了

你权限修改成了什么，我试试

**Reply by 0honus0** [reply]
我找不到文件夹了，大概就是用户组 读写执行，加上把用户组切为root，不知道后面这个需要不，但确实是可以用了

**Reply by jinbolin227** [reply]
> 我找不到文件夹了，大概就是用户组 读写执行，加上把用户组切为root，不知道后面这个需要不，但确实是可以用了

![Screenshot_20210920-070208](https://user-images.githubusercontent.com/56496767/133945738-852f9fe0-44ac-4e49-aec4-cb69b2e1a2ac.jpg)
![Screenshot_20210920-070202](https://user-images.githubusercontent.com/56496767/133945740-504f918d-1bbc-4089-b75a-607c7a1ffd8a.jpg)
还是不行，导出不了文件，

**Reply by 0honus0** [reply]
> > 我找不到文件夹了，大概就是用户组 读写执行，加上把用户组切为root，不知道后面这个需要不，但确实是可以用了
> 
> ![Screenshot_20210920-070208](https://user-images.githubusercontent.com/56496767/133945738-852f9fe0-44ac-4e49-aec4-cb69b2e1a2ac.jpg)
> ![Screenshot_20210920-070202](https://user-images.githubusercontent.com/56496767/133945740-504f918d-1bbc-4089-b75a-607c7a1ffd8a.jpg)
> 还是不行，导出不了文件，

![Screenshot_20210920-111850](https://user-images.githubusercontent.com/63273720/133954050-fc762b33-b8a8-48e2-a95a-a61cd80b2dc7.png)
![Screenshot_20210920-111854](https://user-images.githubusercontent.com/63273720/133954061-991e3680-4717-4865-bdb7-45906c7e8d38.png)
试下这样

**Reply by 0honus0** [reply]
> > 我找不到文件夹了，大概就是用户组 读写执行，加上把用户组切为root，不知道后面这个需要不，但确实是可以用了
> 
> ![Screenshot_20210920-070208](https://user-images.githubusercontent.com/56496767/133945738-852f9fe0-44ac-4e49-aec4-cb69b2e1a2ac.jpg)
> ![Screenshot_20210920-070202](https://user-images.githubusercontent.com/56496767/133945740-504f918d-1bbc-4089-b75a-607c7a1ffd8a.jpg)
> 还是不行，导出不了文件，

设置的时候应用到子目录

**Reply by jinbolin227** [reply]
> > > 我找不到文件夹了，大概就是用户组 读写执行，加上把用户组切为root，不知道后面这个需要不，但确实是可以用了
> > 
> > 
> > ![Screenshot_20210920-070208](https://user-images.githubusercontent.com/56496767/133945738-852f9fe0-44ac-4e49-aec4-cb69b2e1a2ac.jpg)
> > ![Screenshot_20210920-070202](https://user-images.githubusercontent.com/56496767/133945740-504f918d-1bbc-4089-b75a-607c7a1ffd8a.jpg)
> > 还是不行，导出不了文件，
> 
> ![Screenshot_20210920-111850](https://user-images.githubusercontent.com/63273720/133954050-fc762b33-b8a8-48e2-a95a-a61cd80b2dc7.png)
> ![Screenshot_20210920-111854](https://user-images.githubusercontent.com/63273720/133954061-991e3680-4717-4865-bdb7-45906c7e8d38.png)
> 试下这样

同样不行，感觉不是文件夹权限问题

**Reply by 0honus0** [reply]
> > > > 我找不到文件夹了，大概就是用户组 读写执行，加上把用户组切为root，不知道后面这个需要不，但确实是可以用了
> > > 
> > > 
> > > ![Screenshot_20210920-070208](https://user-images.githubusercontent.com/56496767/133945738-852f9fe0-44ac-4e49-aec4-cb69b2e1a2ac.jpg)
> > > ![Screenshot_20210920-070202](https://user-images.githubusercontent.com/56496767/133945740-504f918d-1bbc-4089-b75a-607c7a1ffd8a.jpg)
> > > 还是不行，导出不了文件，
> > 
> > 
> > ![Screenshot_20210920-111850](https://user-images.githubusercontent.com/63273720/133954050-fc762b33-b8a8-48e2-a95a-a61cd80b2dc7.png)
> > ![Screenshot_20210920-111854](https://user-images.githubusercontent.com/63273720/133954061-991e3680-4717-4865-bdb7-45906c7e8d38.png)
> > 试下这样
> 
> 同样不行，感觉不是文件夹权限问题

额，我这样设置之后可以了
你可以尝试修改下给的那些样例的，修改下看保存之后是否生效
